scripts.py

```diff
@@ -517,32 +590,50 @@ def do_generate(self, lane):
         )
         chosen_collections = self.collections or [default_collection]
 
-        for order in chosen_orders:
-            if order not in allowed_orders:
-                logging.warn("Ignoring unsupported ordering %s" % order)
+        for entrypoint_name in chosen_entrypoints:
+            entrypoint = EntryPoint.BY_INTERNAL_NAME.get(entrypoint_name)
+            if not entrypoint:
+                logging.warn("Ignoring unknown entry point %s" % entrypoint_name)
```

Comment:
should you also check that a user-specified entry point is one of the library's enabled entry points?